[2023-05-11T12:30:38.257+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test1.read_and_send_task manual__2023-05-11T06:40:14.417666+00:00 [queued]>
[2023-05-11T12:30:38.264+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test1.read_and_send_task manual__2023-05-11T06:40:14.417666+00:00 [queued]>
[2023-05-11T12:30:38.264+0530] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-11T12:30:38.280+0530] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): read_and_send_task> on 2023-05-11 06:40:14.417666+00:00
[2023-05-11T12:30:38.284+0530] {standard_task_runner.py:57} INFO - Started process 7625 to run task
[2023-05-11T12:30:38.289+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'test1', 'read_and_send_task', 'manual__2023-05-11T06:40:14.417666+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/test1.py', '--cfg-path', '/tmp/tmpfoisbk73']
[2023-05-11T12:30:38.290+0530] {standard_task_runner.py:85} INFO - Job 87: Subtask read_and_send_task
[2023-05-11T12:30:38.337+0530] {task_command.py:410} INFO - Running <TaskInstance: test1.read_and_send_task manual__2023-05-11T06:40:14.417666+00:00 [running]> on host nizam-HP-15-Notebook-PC
[2023-05-11T12:30:38.401+0530] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test1' AIRFLOW_CTX_TASK_ID='read_and_send_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-11T06:40:14.417666+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-11T06:40:14.417666+00:00'
[2023-05-11T12:30:38.426+0530] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2023-05-11T12:30:38.427+0530] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2023-05-11T12:30:38.428+0530] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Connection complete.
[2023-05-11T12:30:38.536+0530] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2023-05-11T12:30:38.536+0530] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2023-05-11T12:30:40.861+0530] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 7625. PIDs of all processes in the group: [7625]
[2023-05-11T12:30:40.861+0530] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 7625
[2023-05-11T12:30:40.862+0530] {taskinstance.py:1540} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-05-11T12:30:40.882+0530] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nizam/airflow/dags/test1.py", line 40, in read_from_mongodb_and_send_to_kafka
    for document in cursor:
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/cursor.py", line 1248, in next
    if len(self.__data) or self._refresh():
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/cursor.py", line 1139, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/mongo_client.py", line 1740, in _ensure_session
    return self.__start_session(True, causal_consistency=False)
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/mongo_client.py", line 1685, in __start_session
    self._topology._check_implicit_session_support()
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/topology.py", line 538, in _check_implicit_session_support
    self._check_session_support()
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/topology.py", line 554, in _check_session_support
    self._select_servers_loop(
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/pymongo/topology.py", line 250, in _select_servers_loop
    self._condition.wait(common.MIN_HEARTBEAT_INTERVAL)
  File "/usr/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
  File "/home/nizam/env_airflow/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1542, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-05-11T12:30:40.895+0530] {taskinstance.py:1368} INFO - Marking task as UP_FOR_RETRY. dag_id=test1, task_id=read_and_send_task, execution_date=20230511T064014, start_date=20230511T070038, end_date=20230511T070040
[2023-05-11T12:30:40.918+0530] {standard_task_runner.py:104} ERROR - Failed to execute job 87 for task read_and_send_task (Task received SIGTERM signal; 7625)
